lib/src/components/NxToast/contexts.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { createContext } from 'react';
+
+import { NxToastContainerContextType } from './types';
+
+const NxToastContainerContext = createContext<NxToastContainerContextType | null>(null);
```

Comment:
If you enforce that the context always has a value, you can remove the nullness check where it's used
```suggestion
const NxToastContainerContext = createContext<NxToastContainerContextType>({ onToastClosing: () => {} });
```

Reply:
> If you enforce that the context always has a value, you can remove the nullness check where it's used

Fixed here: [3ebe47d](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/3ebe47d9523fbb45c7791acfec77b2b874549565)